pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -40,7 +40,7 @@ type ClaimInfo struct {
 // claimInfoCache is a cache of processed resource claims keyed by namespace/claimname.
 type claimInfoCache struct {
 	sync.RWMutex
-	state     state.CheckpointState
+	state     state.Checkpointer
```

Comment:
is it too invasive to rename this variable to `checkpointer`?

Reply:
done